api/environments/views.py

```diff
@@ -203,6 +205,13 @@ def user_permissions(self, request, *args, **kwargs):
     def get_document(self, request, api_key: str):
         return Response(Environment.get_environment_document(api_key))
 
+    @swagger_auto_schema(request_body=no_body, responses={202: ""})
+    @action(detail=True, methods=["POST"], url_path="enable-v2-versioning")
+    def enable_v2_versioning(self, request: Request, api_key: str) -> Response:
+        environment = self.get_object()
```

Comment:
This method is not idempotent, right? We will have multiple initial feature versions if the user makes the call multiple times(I don't think it breaks the code - but it does, create extra data) ?

Reply:
Yep, good catch! Resolved.